api/src/main/java/org/asynchttpclient/BoundRequestBuilder.java

```diff
@@ -93,6 +97,16 @@ public BoundRequestBuilder setBody(String data) {
     }
 
     @Override
+    public BoundRequestBuilder setBody(Publisher<ByteBuffer> publisher) {
+        return super.setBody(new ReactiveStreamBodyGenerator(client.getConfig().getExecutorService(), publisher));
+    }
+
+    @Override
+    public BoundRequestBuilder setBody(BodyGenerator bodyGenerator) {
+        return super.setBody(bodyGenerator);
+    }
```

Comment:
This isn't needed to be overridden since it just delegates straight to super is it?